source/includes/data-formats/time-series.kt

```diff
@@ -0,0 +1,53 @@
+package org.example
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.client.model.*
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.json.JsonWriterSettings
+import java.util.Date
+import org.bson.Document
+
+fun main() {
+    val uri = "<connection string URI>"
+
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString(uri))
+        .retryWrites(true)
+        .build()
+
+    val mongoClient = MongoClient.create(settings)
+
+    // start-create-time-series
+    val database = mongoClient.getDatabase("fall_weather")
+
+    val timeSeriesOptions = TimeSeriesOptions("timestamp")
+    val collectionOptions = CreateCollectionOptions().timeSeriesOptions(timeSeriesOptions)
```

Comment:
S: it is a little confusing to have a variable named timeSeriesOptions as well as a method, I recommend changing one of them to make it clearer